python/cog/server/response_throttler.py

```diff
@@ -0,0 +1,22 @@
+import time
+import os
+
+from ..response import Status
+
+
+class ResponseThrottler:
+    def __init__(self):
+        self.last_sent_response_time = 0
+        self.response_interval = float(
+            os.environ.get("COG_THROTTLE_RESPONSE_INTERVAL", 0)
```

Comment:
I feel like this should be passed in as a parameter rather than pulled from the environment. I can imagine wanting to do this at predict time, to be more reactive to managing a cluster of models without having to reload the containers to get a new environment variables. In that situation passing it in makes it more obvious how/where that fit in.